paraatm/tests/aviationr_example.py

```diff
@@ -0,0 +1,52 @@
+import os
+import numpy as np
+import pandas as pd
+from paraatm.simulation_method.aviationr import AviationRisk
+import matplotlib.pyplot as plt
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+data_dir = os.path.join(cur_dir, '..', 'sample_data/')
+os.environ["NATS_HOME"] = '/home/xinyu/Project/NATS'
```

Comment:
Please remove reference to your local `NATS_HOME` settings.  You can set this environment variable directly on your system.  On Linux, you can add `export NATS_HOME=/path/to/NATS` to your `~/.bashrc` file.

Reply:
Yeah. I will update it soon. For the model size, what size do you think is proper? I have limited risk model from 20mb to 1mb. Will it be OK? For the deep learning model, it might be a little bit hard to reduce the model size since the architecture is fixed. I am still trying to find a way to do it.